src/jimctrl_init_helper_clear_80037FB8.c

```diff
@@ -0,0 +1,16 @@
+#include <util/idaTypes.h>
+#include "linker.h"
+#include "jimctrl.h"
+
+extern array_800B9340_child array_800B9340[array_800B9340_SIZE];
+
+void jimctrl_init_helper_clear_80037FB8(void)
+{
+    array_800B9340_child *puVar1;
+    int iterator = 5;
+    puVar1 = &array_800B9340[0];
```

Comment:
This line seems a bit iffy, but it doesn't work out without it, or with it being set to just 5 from the get-go (with nothing in the for declaration section)